Gems/EMotionFX/Code/EMotionFX/Source/MotionInstance.h

```diff
@@ -20,6 +20,9 @@
 #include <EMotionFX/Source/EventInfo.h>
 #include <EMotionFX/Source/MotionInstancePool.h>
 
+#if defined GetCurrentTime
+#undef GetCurrentTime
+#endif
```

Comment:
Why undef this here?

Reply:
The change in the headers for the Profiler.h somehow caused this macro to get defined from a windows header elsewhere. The undef in the profiler doesn't actually catch this so some other header is bringing it in and "infecting" the macro namespace as it were.